lms/views/dashboard.py

```diff
@@ -62,9 +74,37 @@ def assignment_show(self):
         Authenticated via the LTIUser present in a cookie making this endpoint accessible directly in the browser.
         """
         assignment = self.get_request_assignment()
-        self.request.context.js_config.enable_dashboard_mode(assignment)
+        self.request.context.js_config.enable_dashboard_mode(
```

Comment:
The config has not changed here but is not built on the view instead of JSConfig.